app/workers/asset_manager_create_whitehall_asset_worker.rb

```diff
@@ -3,7 +3,26 @@ class AssetManagerCreateWhitehallAssetWorker < WorkerBase
 
   sidekiq_options queue: "asset_manager"
 
-  def perform(file_path, legacy_url_path, draft = false, attachable_model_class = nil, attachable_model_id = nil, auth_bypass_ids = [])
+  WITH_TEMPORARY_ASSET_RELATED_PARAMS = 8
+  def perform(*args)
+    # We eventually only run the code thats inside "perform_with_assets_params"
+    # these changes are introduced to fix a prod issue that was caused by change in
+    # arguments( decreased from 8 to 6 ) that were accepted by perform method
+    # these cause one of the job that was about to execute just before application deployment
+    # to fail. the changes are temporary.
+
+    if args.length < WITH_TEMPORARY_ASSET_RELATED_PARAMS
```

Comment:
Minor thing - I'd prefer to have explicit equality checks, something like
```
if args.length == EXPECTED_PARAMS
...
else if args.length == WITH_TEMPORARY_ASSET_RELATED_PARMS
...
else raise "invalid parameter count in sidekiq job"
```
I guess that might be tricky with optional parameters though.  But I wonder - why do we have optional parameters? Are they ever not passed???

anyway, as a temporary change that will last only days, this isn't vital.

Reply:
incorporated these changes. having explicit equality checks made more sense.